presto-native-execution/presto_cpp/main/operators/PartitionAndSerialize.cpp

```diff
@@ -111,6 +131,53 @@ class PartitionAndSerializeOperator : public Operator {
   }
 
  private:
+  void prepareOutput(vector_size_t size) {
```

Comment:
size might not be required as we directly access input_ in class member method? Thanks!